development-guidelines.md

```diff
@@ -57,3 +57,11 @@ workflow:
 
 Always check the feedback that these tools automatically provide to each
 Pull Request and implement the changes necessary or suggested.
+
+
+### Third-party libraries
+
+IO project uses several 3th part open source libraries. When including or updating a library could happen that the library's code should be fixed. In that case two options can be evaluated:
```

Comment:
```suggestion
IO includes several third-party open source libraries in its components. When including or updating a library it could happen that the library's code needs some changes (e.h. bugfixes). In that case two options can be evaluated:
```